cg_lims/EPPs/udf/copy/qc_to_sample.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import sys
+
+import click
+from genologics.entities import Process
+from genologics.lims import Lims
+
+from cg_lims import options
+from cg_lims.exceptions import LimsError, MissingUDFsError
+from cg_lims.get.artifacts import get_artifacts
+
+
+def artifacts_to_sample(
+    artifacts: list, sample_qc_udf: str = None
+) -> None:
+
+    """Function to copy artifact qc flag to sample level.
+
+    For each artifact in the artifacts list, copying the qc_flagg
+    to all samples related to the artifact. If a art is a pool, art.samples
+    is a list with many samples. Otherwiese a list with only one sample.
+
+    Arguments:
+        artifacts: list of artifacts to copy from
+        sample_qc_udf: sample qc udf to set based on artifact qc_flag"""
+
+    failed_udfs = 0
+    passed_udfs = 0
+    for art in artifacts:
```

Comment:
```suggestion
    for artifact in artifacts:
```

replace art with artifact at all places below